packages/test/test-end-to-end-tests/src/test/data-virtualization/groupIdOffline.spec.ts

```diff
@@ -326,8 +326,15 @@ describeCompat("GroupId offline", "NoCompat", (getTestObjectProvider, apis) => {
 		// There are two parts to refresh, making the network snapshot call and trimming the ops
 		// Container layer Refresh
 		// Network call refreshing the base snapshot
-		const serializedStateManager = (container2 as any).serializedStateManager;
-		await serializedStateManager.refreshLatestSnapshot();
```

Comment:
No param here caused the groupIds code to be skipped. When I pass true the test fails. Skipping for now - @tyler-cai-microsoft want me to open a bug for you to follow up?

Reply:
Yes, please do

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/7996